prototyping/test_clibrary.py

```diff
@@ -0,0 +1,31 @@
+import ctypes
+from ctypes import CDLL
+from ctypes.util import find_library
+
+import pytest
+
+from src.compiler import compile_lib, SRC
+from src.structs import Point
+
+lib_filename = find_library("clibrary")
+lib = CDLL(lib_filename)
+
+
+@pytest.fixture
+def wrapper():
+    source = SRC / "clibrary.cpp"
+    compile_lib(source)
```

Comment:
Alternatively, we can provide the path here using the `cflags` optional argument.